pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -530,6 +540,15 @@ func (l *LibvirtDomainManager) asyncMigrate(vmi *v1.VirtualMachineInstance, opti
 	}(l, vmi)
 }
 
+func isDomainPaused(dom cli.VirDomain) (bool, error) {
+    status, reason, err := dom.GetState()
+    if err != nil {
+        return false, err
+    }
+    return util.ConvState(status) == api.Paused &&
+            util.ConvReason(status, reason) == api.ReasonPausedUser, nil
```

Comment:
What is the reason for `== api.ReasonPausedUser`

Reply:
We set VMI condition based on this, here: https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-handler/vm.go#L1075

Reply:
I know I just have this addition to it https://github.com/kubevirt/kubevirt/pull/5401 :) So I was wondering if we want to migrate pause only for a specific reason...

Reply:
Ah, well, I think this is the only case we should care about keeping the VMI paused after the migration.
In general, libvirt should add this flag to any paused domain, but I just wanted to explicitly add it here in case something changes in libvirt or it gets into a weird flow.

Reply:
https://github.com/libvirt/libvirt/blob/master/src/qemu/qemu_migration.c#L4795